### ISSUE TITLE
Explore aligning accessibility testing plan with the accessibility component checklist

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#87897]
Assignee: @briandeconinck @rsmithadhoc 
Peer Reviewer:

## Description
The VADS components that are categorized as USE: Best Practice per the [Maturity Scale ](https://design.va.gov/about/maturity-scale) contain a Component Checklist at the bottom of the VADS component guidance. Within the Component Checklist is the Accessibility Component Checklist. The current accessibility component checklist contains 5 sections for Accessible use of color, Accessible contrast, Keyboard interactions, Content zoom tested, and Tested in screen readers.

The purpose of this ticket is to align the accessibility checklist with what is currently documented in the Accessibility testing plan.

In addition, we would also like to create a template or framework for communicating how a VADS component aligns to the categories within the Accessibility Checklist. This will be used post Staging Reviews so that DST may update the guidance on VADS with regards to how the component tested. Currently, the accessibility checklist on VADS only has the option to pass the category, or fail the category. If it is necessary to communicate at a more granular level, this ticket will also explore how we might do that. 

Notes from 7/9: how can we automate this to make it easy for DST to update the component checklist pages? Possibly also explore the component last audit date, how many changes have been down since then, then creating gh issues automatically to reaudit those components. 

## Tasks
- [ ] _Placeholder task_

## Peer Review
_To be completed by peer reviewer_
- [ ] User story and acceptance criteria are met

## Acceptance Criteria
- [ ] We have identified how the categories in the accessibility checklist will be modified to align with the accessibility testing plan.
- [ ] We have outlined a template for DST and Governance to complete at a component Staging Review so that the accessibility checklist on VADS can be easily updated

## Team Notification
- [ ] Slack post on #platform-governance-team
- [ ] Presentation at Weekly Initiative Meeting
- [x] No notification needed

### Refinement
- [x] Ticket has user story, description, tasks, and acceptance criteria
- [x] Ticket has been sized 
- [x] Once all above items are checked, add the 'Ready' label.
### Planning
_If this ticket is picked up from the Backlog mid-sprint, connect with Shira to ensure the below items are completed correctly_
- [x] Ticket has been discussed as a team at Planning
- [x] Ticket has been assigned to appropriate initiative or quarterly epic
- [x] Ticket has been assigned to a Sprint
- [ ] Ticket has been moved to the Planned pipeline in the Governance team board
- [ ] Ticket as an assignee and peer reviewer

**Comment by rsmithadhoc** [comment]
@briandeconinck I have a PR mentioned in the comment above if you would like to take a look. I don't believe you need to code review it (unless you want to and are familiar with Jekyll), but any feedback on entering audit data, how it's displayed, and the instructions would be good.

**Comment by caw310** [comment]
Closing as Ryan's work on this is done.
